chia/server/node_discovery.py

```diff
@@ -272,10 +272,18 @@ async def _connect_to_peers(self, random: Random) -> None:
         next_feeler = self._poisson_next_send(time.time() * 1000 * 1000, 240, random)
         retry_introducers = False
         introducer_attempts: int = 0
```

Comment:
it doesn't look like we need this counter anymore

Reply:
Thanks